app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/repository/RepositoryBrowserViewModel.kt

```diff
@@ -18,8 +19,10 @@ class RepositoryBrowserViewModel : BaseViewModel() {
     private val repo: BitbucketRepository by inject()
 
     // State
-    private val srcFiles = MutableStateFlow<List<RepoFile>>(emptyList())
-    private var currentRepoName: String = ""
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
I know that private is the default setting, but I add the otherwise value to prevent any confusion